### DISCUSSION TITLE
We repeatedly got the Graphql exception in exception.log

### DISCUSSION THREAD

**Discussion by lakshmisasanka** [webonyx/graphql-php#1213] [Q&A]

**Comment by spawnia** [comment]
This is not an issue with this library or your server, you simply have a client sending faulty requests. The only thing I would fix is to stop logging client-safe exceptions.

**Comment by lakshmisasanka** [comment]
Hi

Could you please let me know the Client who is sending the faulty request
or IP address?


Regards,
Sasanka.J

**Reply by spawnia** [reply]
I am confused by what you are asking, this problem happened in your environment. This project is just a library, not a personal helpdesk for issues you are having with your application.
